spec/controllers/v0/terms_of_use_agreements_controller_spec.rb

```diff
@@ -116,6 +116,20 @@
         end
       end
 
+      context 'when the user does not have a common_name' do
+        let(:expected_error) { 'Name for user must be present' }
+
+        before do
+          allow_any_instance_of(User).to receive(:common_name).and_return(nil)
```

Comment:
Using `allow_any_instance_of` is generally [discouraged](http://rspec.info/documentation/3.12/rspec-mocks/#settings-mocks-or-stubs-on-any-instance-of-a-class). You could probably just set `common_name` to `nil` when you call the user factory for this example.

Reply:
I'm not sure why there's a `common_name` attribute in the Users factory, it's not an attribute but a method that dynamically generates the string:
```ruby
# models/user.rb
  def common_name
    [first_name, middle_name, last_name, suffix].compact.join(' ')
  end
```

I tried using a new factory declaration (which already defaults `common_name` to nil) but it doesn't seem to have any effect.

Reply:
Okay I just updated the user factory declaration to have all of the component names be `nil` and that seems to have done the trick.

`let(:user) { create(:user, first_name: nil, middle_name: nil, last_name: nil, suffix: nil) }`